### PR TITLE
Migrate Devise::DeviseAuthyController#request_sms

### PR DESCRIPTION
**Pull request by wwahammy**

### CODE REVIEW

File: app/controllers/devise/devise_authy_controller.rb [hunk 1]
```diff
@@ -172,13 +172,16 @@ def request_phone_call
...
-    if !@resource
-      render :json => {:sent => false, :message => "User couldn't be found."}
+    unless @resource
+      render json: { sent: false, message: "User couldn't be found." }
       return
     end
 
-    response = Authy::API.request_sms(:id => @resource.authy_id, :force => true)
-    render :json => {:sent => response.ok?, :message => response.message}
+    mfa_config = @resource.mfa_config
```
Comment (BintLopez) [inline app/controllers/devise/devise_authy_controller.rb:180]: Do we need to handle the case where `mfa_config` is `nil`? (Is that even a feasible scenario?)

Comment (wwahammy) [reply app/controllers/devise/devise_authy_controller.rb:180]: We [don't currently in piggybank](https://github.com/MidwestAccessCoalition/cashbox/blob/f873c1c7a0e870cf7385fe84f8531d4b38b3b428/app/controllers/users/devise_authy_controller.rb#L102C3-L114C1) but that's a good catch! I'll add that

Comment (wwahammy) [reply app/controllers/devise/devise_authy_controller.rb:180]: Updated!

**Review by BintLopez — APPROVED** [review]
🦑

**Pull request merged**